site/guide/configuration/configure-azure-private-link.qmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "Configure Azure Private Link"
+date: last-modified
+aliases:
+  - ../configure-aws-privatelink.html
+---
+
+To keep your network traffic private and minimizes its attack surface, configure Azure Private Link[^1] to establish a private connection between ValidMind and your company network.
```

Comment:
> To keep your network traffic private and minimizes its attack surface,

I can't really tell what the second half of this sentence is getting at — is it "to minimize its attack surface"? What is "it" referring to in this case (antecedent unclear, or maybe my brain is just dumb)?

Reply:
It's a broken sentence I had cribbed from the AWS PrivateLink topic. Should have been: "To keep your network traffic private and minimize its attack surface ..." — now fixed in both places!